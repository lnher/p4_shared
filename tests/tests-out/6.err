expect: spawn id exp5 not open
    while executing
"expect "$ ""
    (file "../tests/run-xv6-command.exp" line 25)
