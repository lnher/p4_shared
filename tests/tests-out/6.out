XV6_TEST_OUTPUT After vmalloc huge, 1MB - program base pages:0 huge pages:1
XV6_TEST_OUTPUT Sanity checking vmalloc address 1E008000
